st7735r.cpp

```diff
@@ -62,14 +62,19 @@ void InitST7735R()
     madctl ^= MADCTL_ROTATE_180_DEGREES;
 #endif
 
+#if defined(ST7789V2) && defined(DISPLAY_OUTPUT_LANDSCAPE)

```

Comment:
This probably should be an unconditional

```c
#if defined(DISPLAY_OUTPUT_LANDSCAPE)
```
Looks like this directive was missing from the driver compared to e.g. ili9341 setup of the madctl register (which should be identical to st77xx behavior of the register)

Reply:
Based on the conversation of panel orientation, leaving this change for the ST7789V2 looks good.